docs/sources/panels-visualizations/visualizations/pie-chart/index.md

```diff
@@ -35,6 +35,21 @@ The following video guides you through the creation steps and common customizati
 
 {{< docs/play title="Grafana Bar Charts and Pie Charts" url="https://play.grafana.org/d/ktMs4D6Mk/" >}}
 
+## Supported data formats
+
+The pie chart panel requires a data set with at least one row with numeric values to be added up. If the first row contains column titles it will be used to display and identify the values in the pie chart. Cells with non-numeric values will not be displayed inside of the pie chart.
```

Comment:
We always call these visualizations vs panels. Panels include a visualization as well as data and any transformations you apply. We wanna be really specific about this going forward. 

```suggestion
The pie chart visualization requires a dataset with at least one row of numeric values to be added together. If the first row contains column titles, it's used to display and identify the values in the pie chart. Cells with non-numeric values are not displayed in the pie chart.
```